### PR TITLE
Adding Fix and Unit tests for UTF-8 Breakages

### PR DESCRIPTION
**Pull request by CalvinChan1**
We’re trying to encode as ascii when the text is probably UTF-8 thus getting errors like: `UnicodeEncodeError: 'ascii' codec can't encode characters in position 133-141: ordinal not in range(128)`

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

📝 Please visit https://github.com/snowplow/snowplow/wiki/CLA to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: snowplow_tracker/__init__.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
 from snowplow_tracker.subject import Subject
-from snowplow_tracker.emitters import logger, Emitter, AsyncEmitter, CeleryEmitter, RedisEmitter
+from snowplow_tracker.emitters import Emitter, AsyncEmitter, CeleryEmitter, RedisEmitter
 from snowplow_tracker.self_describing_json import SelfDescribingJson
 from snowplow_tracker.tracker import Tracker
 from contracts import disable_all as disable_contracts, enable_all as enable_contracts
+import logging
+
+# Set default logging handler to avoid "No handler found" warnings.
+logging.getLogger(__name__).addHandler(logging.NullHandler())
```
Comment (BenFradet) [inline snowplow_tracker/__init__.py]: why change the existing logging?

File: snowplow_tracker/emitters.py [hunk 2]
```diff
@@ -52,16 +51,6 @@
...
 
-try:
-    # Check whether a custom Celery configuration module named "snowplow_celery_config" exists
-    import snowplow_celery_config
-    app = Celery()
-    app.config_from_object(snowplow_celery_config)
-
-except ImportError:
-    # Otherwise configure Celery with default settings
-    app = Celery("Snowplow", broker="redis://guest@localhost//")
```
Comment (BenFradet) [inline snowplow_tracker/emitters.py]: what was the motivation behind this?

File: snowplow_tracker/payload.py [hunk 3]
```diff
@@ -78,15 +78,15 @@ def add_json(self, dict_, encode_base64, type_when_encoded, type_when_not_encode
         """
 
         if dict_ is not None and dict_ != {}:
-
             json_dict = json.dumps(dict_, ensure_ascii=False)
-
             if encode_base64:
-                encoded_dict = base64.urlsafe_b64encode(json_dict.encode("ascii"))
```
Comment (bernardosrulzon) [inline snowplow_tracker/payload.py]: This PR doesn't solve the issue https://github.com/snowplow/snowplow-python-tracker/issues/194 in my local tests

Simply changing the encoding on this line to `utf-8` does the trick

**Comment by oguzhanunlu** [comment]
Closing due to long inactivity & missing CLA sign. This issue will be solved by #219 .

**Pull request closed without merging**